module/renderer/src/androidUnitTest/kotlin/org/cru/godtools/shared/renderer/content/RenderButtonPaparazziTest.kt

```diff
@@ -114,4 +117,22 @@ class RenderButtonPaparazziTest : BasePaparazziTest() {
             modifier = Modifier.fillMaxWidth()
         )
     }
+
+    @Test
+    fun `RenderButton() - IsInvisible`() = contentSnapshot {
+        RenderContentStack(
+            listOf(
+                Text(text = "Before"),
+                Button(
+                    style = Button.Style.CONTAINED,
+                    width = Dimension.Percent(0.5f),
+                    gravity = Gravity.Horizontal.CENTER,
+                    text = { Text(it, "IsInvisible Button") },
+                    goneIf = "isSet(a)"
+                ),
+                Text(text = "Before")
+            ),
+            modifier = Modifier.fillMaxWidth()
```

Comment:
you will need to pass a state object into `RenderContentStack`. you will probably want to create it outside of the `contentSnapshot` closure